test/access_control.sh

```diff
@@ -119,9 +151,7 @@ function run_curl_cmd() {
 function test_user_has_proper_access() {
     local user=$1; shift
     local pw=$1; shift
-    # rest - indices to which access should be granted, followed by --,
-    # followed by indices to which access should not be granted
-    local expected=1
+    local expected=2
```

Comment:
why do we still set `expected` ? it doesn't look like we still use this variable